Hearthstone Deck Tracker/Config.cs

```diff
@@ -732,9 +738,11 @@ public class Config
 		[DefaultValue("enUS")]
 		public string SelectedLanguage = "enUS";
 
-		[XmlArray(ElementName = "AlternativeLanguages")]
-		[XmlArrayItem(ElementName = "Language")]
-		public List<string> AlternativeLanguages = new List<string>();
+		[DefaultValue(false)]
+		public bool LargeCardImages = false;
```

Comment:
Let's turn this into a `double` with `DefaultValue(1)` and add a dropdown or slider (depending on what is easier) to the settings instead of the checkbox with `1x, 1.25x, 1.5x, 1.75x, 2x`.

With a slider I believe you can set `min=1, max=2, step=0.25`